golem_messages/utils.py

```diff
@@ -2,9 +2,21 @@
 import copy
 from uuid import UUID
 
+from eth_utils import to_checksum_address
+from ethereum.utils import sha3
+
 from golem_messages import message
 
 
+def pubkey_to_address(pubkey: str) -> str:
```

Comment:
there at least two available implementations of this available:
* in bitcoin that is already a dep for GM [src](https://github.com/vbuterin/pybitcointools/blob/aeb0a2bbb8bbfe421432d776c649650eaeb882a5/bitcoin/main.py#L450)
* in pyethereum [src](https://github.com/ethereum/pyethereum/blob/b704a5c6577863edc539a1ec3d2620a443b950fb/ethereum/utils.py#L119)

maybe you can use it...

Reply:
Bitcoin implementation returns a Bitcoin address - not what we want
this pyethereum implementation that you linked returns an address of a contract that will be created, not something that applies here
Last time I checked there was no implementation of this kind, which is weird, but that's the case.